src/pages/repository.html

```diff
@@ -1,297 +1,272 @@
+<link href="css/repositories.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet">
 
+<link href="css/lib/bootstrap-switch/bootstrap-switch.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet">
 
-	<link href="css/repositories.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet">
+<div id="wb-repository">
+    <h1 ng-if="repositoryType === repositoryTypes.free">Create GraphDB Free repository</h1>
```

Comment:
Why don't you just resolve the label in the controller instead of all these ng-if's?!?